tools/build-tools/data/layerInfo.json

```diff
@@ -152,7 +152,8 @@
                 "dev": true,
                 "packages": [
                     "@fluid-experimental/get-container",
-                    "@fluid-experimental/fluid-static"
+                    "@fluid-experimental/fluid-static",
+                    "@fluid-experimental/tinylicious-client"
```

Comment:
layer-check is a two stage change.
1. Check into layer check
2. wait for new build-tools to publish
3. consume build-tools pkg from root package-json and submit.

Reply:
Will follow-up on this change with the additional PR to bump the build-tools version